app/src/tiles/dataDefinition.ts

```diff
@@ -7,12 +7,16 @@ const LAYER_QUERIES = {
         SELECT
             geometry_id
         FROM
-            buildings`,
+            buildings
+        WHERE
+            latest_demolish_date IS NULL`,
```

Comment:
Hi @tomalrussell - I'm at the point where I have scripts that I can use to update the postgres db which I'm ready to try out on the staging server - one thing I still haven't correctly implemented is a change that will prevent buildings from the buildings table showing up when their `latest_demolish_date` is anything other than `NULL` (if you remember our discussion, that's the simple check we're going with for the time being). I've made an attempt here to update the ts code in the right place after looking at your explanation on #800 - this doesn't seem to be sufficient to prevent a building in the buildings table showing up in the app when is has a `latest_demolish_date` - any thoughts what else I need to do?

Reply:
Thanks Tom, that's really helpful, I think this almost works correctly now - you were right it was doing something anyway, but now should be more general with the change I've made. I'm still getting the geometry id from "demolished" buildings showing up when I click on where they're located though. I'll investigate further, but this might not matter, so long as the polygon is non-editable in the website.

Reply:
That goes through the API `/api/buildings/locate?lng=...&lat=...`

from the frontend, the request is made in response to a map click:
- https://github.com/colouring-cities/colouring-london/blob/e12b544503a5e5ba302387a8c2bddee1a5be514d/app/src/frontend/map/map.tsx#L60

from the backend, here's the stack drilling down to where that is eventually handled in a database query: 
- entry point https://github.com/colouring-cities/colouring-london/blob/master/app/src/api/api.ts#L20
- server mounts API https://github.com/colouring-cities/colouring-london/blob/master/app/src/server.tsx#L47
- buildings routes loaded at https://github.com/colouring-cities/colouring-london/blob/master/app/src/api/api.ts#L20
- route defined at https://github.com/colouring-cities/colouring-london/blob/master/app/src/api/routes/buildingsRouter.ts#L12
- controller method at https://github.com/colouring-cities/colouring-london/blob/master/app/src/api/controllers/buildingController.ts#L16
- query at https://github.com/colouring-cities/colouring-london/blob/a4771eaac0717febee4e1e57b585ac6aca84c9e5/app/src/api/services/building/query.ts#L3

Reply:
Brilliant thanks, editing query.ts did the trick